### PR TITLE
archivers: create lz4 and alias profiles

### PR DESCRIPTION
**Pull request by glitsj16**
Follow-up to #5656.
Partly relates to #5610.

### CODE REVIEW

File: etc/profile-a-l/lz4.profile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Description: Compress or decompress .lz4 files
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include lz4.local
+# Persistent global definitions
+include globals.local
+
+# mkinitcpio compression support
+private-etc mkinitcpio*
```
Comment (kmk3) [inline etc/profile-a-l/lz4.profile:11]:
```suggestion
```

Same as the following comment:

* <https://github.com/netblue30/firejail/pull/5656#pullrequestreview-1294491046>

Comment (glitsj16) [reply etc/profile-a-l/lz4.profile]: Fixed, thx!

**Comment by glitsj16** [comment]
@kmk3 Question: nothing super urgent, but if 0.9.74 is arriving I wonder how I could best proceed on this. I won't touch anything yet, incl. syncing and resolving conflicts. I admit I totally forgot this draft until today, upstreaming several new profiles I have been using for quite a while (as you can see in the recent PR's). TIA.

**Review by kmk3 — COMMENTED** [review]
> @glitsj16 glitsj16 force-pushed the lz4 branch from add09bb to e9b730e [on
> Mar 3, 2023](https://github.com/netblue30/firejail/pull/5657#event-8660486976)

I don't think I had seen this, but it looks like it worked as intended.

@glitsj16 [on Mar 1](https://github.com/netblue30/firejail/pull/5657#issuecomment-1973130143):

> Question: nothing super urgent, but if 0.9.74 is arriving I wonder how I
> could best proceed on this. I won't touch anything yet, incl. syncing and
> resolving conflicts. I admit I totally forgot this draft until today,
> upstreaming several new profiles I have been using for quite a while (as you
> can see in the recent PR's). TIA.

The PR looks mostly fine but it is a bit outdated; can you try to rebase to the
current master (and update the comments to be in the current `# lz4 # [...]`
format)?

And if possible remove the RELNOTES changes (in this case it can be
auto-generated later).

Also, the PR adds all of the new programs except for `lz4c` to firecfg.config,
is this intentional?

**Comment by kmk3** [comment]
(Moved to #6241)

**Pull request closed without merging**